src/main/webapp/WEB-INF/themes/desktop/sass/pages/_article.scss

```diff
@@ -411,6 +411,9 @@ $article-sidebar-width: 160;
     a.article-tab-5 {
       background: lighten($tabs-base-color, 40%);
     }
+    a.article-tab-6 {
+      background: lighten($tabs-base-color, 30%);
```

Comment:
Shouldn't this be 50%?

Reply:
I couldn't remember if we were taking the mockup literally with those shades.  It can be 50%